docs/source/asr/models.rst

```diff
@@ -285,6 +285,31 @@ You may find the example config files of Conformer variant of such hybrid models
 with sub-word encoding at ``<NeMo_git_root>/examples/asr/conf/conformer/hybrid_transducer_ctc/conformer_hybrid_transducer_ctc_bpe.yaml``.
 
 
+.. _Conformer-HAT_model:
+
+Conformer-HAT (Hybrid Autoregressive Transducer)
+--------------------------------------
+Conformer HAT model (do not confuse it with Hybrid-Transducer-CTC) is a modification of Conformer-Transducer model based on `Google paper <https://arxiv.org/abs/2003.07705>`_.
+The main idea is to separate labels and blank score predictions, which allows to estimate the internal LM probabilities during decoding.
+When external LM is available for inference, the internal LM can be subtracted from HAT model prediction in beamsearch decoding to improve external LM efficiency.
+It can be helpful in the case of text-only adaptation for new domains.
```

Comment:
How can users use this feature?
Do the current LM scripts support it?

Reply:
By default Conformer HAT model works in decoding time as a standard Transducer model with the same interface. However, if you have an external ngram LM you can use `scripts/asr_language_modeling/ngram_lm/eval_beamsearch_ngram_transducer.py` script. The new updated version of the script is under reviewing -- https://github.com/NVIDIA/NeMo/pull/6370

Reply:
@VahidooX -- could you approve the PR if everything is OK?